pymatgen/io/aims/parsers.py

```diff
@@ -633,7 +639,7 @@ def forces(self) -> list[Vector3D] | None:
         )
 
     @property
-    def stresses(self) -> list[Matrix3D] | None:
+    def stresses(self) -> Sequence[Matrix3D] | None:
```

Comment:
Better not use sequence here. best to be as generic as possible in arguments and as specific as possible in return values.

Reply:
This is fixed now, but I am not sure what caused the CI errors for VASP